styles.css

```diff
@@ -227,7 +229,7 @@ form button {
 #see-project-small {
   width: 18.5vh;
   height: 7.3vh;
-  margin: 0 5vh;
+  margin-left: 0%;
   font-size: 2.58vh;
 }
```

Comment:
- [ ] Kindly correct the padding of your "See Project" on the desktop version.
![image](https://user-images.githubusercontent.com/61919617/171907737-5a416d24-b6cf-4f23-8b34-ad2012563d59.png)